### DISCUSSION TITLE
QEMU Guest Agent missing in generic-aarch64

### DISCUSSION THREAD

**Discussion by imjuzcy** [home-assistant/operating-system#3445] [Ideas]
I'm running the `generic-aarch64` image in QEMU on a Raspberry Pi, but it seems that QEMU Guest Agent is not in the image. 

This makes the VM lack integration and features to monitor the VM from the hypervisor, and also not have memory ballooning, which in a memory limited board it's favourable to have because I would prefer that the VM not hog the memory even when it's not needed.

**Comment by Baxxy13** [comment]
It should be there... #2213

**Reply by imjuzcy** [reply]
Indeed. It was my mistake to not check the logs in the VM for it. The cause for the problem with memory ballooning lies somewhere else that I have to find then.
